enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,323 @@
+
+## Design Details
+
+### Open Questions [optional]
+
+N/A
+
+### Test Plan
+TBD
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+N/A
+
+#### Tech Preview -> GA
+N/A
+
+#### Removing a deprecated feature
+N/A
+
+### Upgrade / Downgrade Strategy
+
+Same as [the component selection strategy](/enhancements/installer/component-selection.md#upgrade--downgrade-strategy).     
+Note that:
+- the capabilities [cannot disabled post install](/enhancements/installer/component-selection.md#capabilities-cannot-be-uninstalled)
+- the capabilities can be [implicitly enabled](/enhancements/installer/component-selection.md#updates) (e.g. cluster version with `None` baseline upgrades to the version which moves some of the core payloads to a capability)
+
+### Version Skew Strategy
+N/A
+
+### Operational Aspects of API Extensions
+N/A - no api extensions are being introduced in this proposal
+
+#### Failure Modes
+N/A - no api extensions are being introduced in this proposal
+
+#### Support Procedures
+N/A - no api extensions are being introduced in this proposal
+
+## Implementation History
+N/A
+
+## Alternatives
+
+### Implement the ingress capability for the standalone OpenShift
+
+The goal of this enhancement is to satisfy the HyperShift's requirements. The implementation of the ingress capability for the standalone OpenShift brings a lot of additional complexity, unnecessary for HyperShift. Most probably the following components would need to be updated to tolerate the absence of the ingress operator:
+- The openshift installer. It [checks for the admission of the console route](https://github.com/wking/openshift-installer/blob/3d49ffbff57011bbf846bf91de11526bb3543a83/cmd/openshift-install/create.go#L543) by the ingress controller.
+- The authentication operator. It checks the availability of [the authentication server's](https://github.com/openshift/cluster-authentication-operator/blob/da39951a53ad95be28e32a48f278cc23f41e99a7/pkg/controllers/oauthendpoints/oauth_endpoints_controller.go#L30-L31) 
+and [console](https://github.com/openshift/cluster-authentication-operator/blob/da39951a53ad95be28e32a48f278cc23f41e99a7/pkg/controllers/configobservation/console/observe_consoleurl.go#L15) routes.
+- [The component routes API](https://docs.openshift.com/container-platform/4.13/rest_api/config_apis/ingress-config-openshift-io-v1.html#status-componentroutes). The ingress operator is [responsible for the RBAC](https://github.com/openshift/cluster-ingress-operator/blob/8f98c618c5609cf7fcb97e5c61dc1e7a30576925/pkg/operator/controller/configurable-route/controller.go#L42-L52)
+which the component operators need to be able to access the custom TLS certificates from `openshift-config` namespace.
+
+**Note**: it's difficult to anticipate the impact of the ingress operator removal on the installation process without the actual capability implementation.
```

Comment:
I think the key point here is that implementing the capability for HyperShift is a useful step towards implementing the capability for standalone OpenShift as well, right?

Reply:
Rephrased to 

> **Note**: it's difficult to anticipate the impact of the ingress operator removal on the installation process without the actual capability implementation. However, implementing the capability for HyperShift is a valuable step towards implementing the capability for standalone OpenShift as well.